src/Whoops/Resources/css/whoops.base.css

```diff
@@ -490,18 +490,15 @@ pre.sf-dump {
+      .search-for-help li a svg.google {
+        fill: #fff;
+      }
+      .search-for-help li a svg.google path {
+        background-size: contain;
+      }
+      .search-for-help li a svg.stackoverflow {
+        fill: #fff;
+      }
+      .search-for-help li a svg.google path {
```

Comment:
This is a duplicate selector and a duplicate body. 6 lines above there’s the same.